tiledb/type/range/range.h

```diff
@@ -163,8 +163,8 @@ class Range {
 
   /** Constructs a range and sets variable data. */
   Range(
-      const std::string& s1,
-      const std::string& s2,
+      const std::string_view& s1,
+      const std::string_view& s2,
```

Comment:
`string_view` should be passed by value, not by reference.

Reply:
@teo-tsirpanis did you forget to push?

Reply:
I have pushed, GitHub says "Outdated" on the diff.

Reply:
🤦‍♂️